fontc/src/main.rs

```diff
@@ -167,69 +166,61 @@ fn glyphs_deleted<'a>(current_inputs: &Input, prev_inputs: &'a Input) -> HashSet
         .collect()
 }
 
-fn add_static_metadata_work(
+fn add_static_metadata_ir_job(
     change_detector: &mut ChangeDetector,
-    context: &Context,
-    work: &mut Vec<Box<dyn Work>>,
+    jobs: &mut HashMap<WorkIdentifier, Job>,
```

Comment:
not blocking, but I'd probably prefer some sort sort of `JobSet` type or something, instead of always passing around the hashmap?

Reply:
Good call, will add.

Reply:
I added a Workload type which is consumed by calling exec. Feels better, good suggestion.